DataFormats/HLTReco/interface/TriggerEventWithRefs.h

```diff
@@ -300,18 +320,42 @@ namespace trigger {
       return std::pair<size_type, size_type>(begin, end);
     }
 
+    std::pair<size_type, size_type> l1tkmuonSlice(size_type filter) const {
+      const size_type begin(filter == 0 ? 0 : filterObjects_.at(filter - 1).l1tkmuon_);
+      const size_type end(filterObjects_.at(filter).l1tkmuon_);
+      return std::pair<size_type, size_type>(begin, end);
```

Comment:
slightly easier to use `make_pair` with types inferred, but I recognize this is a pattern already existing in this code